main.go

```diff
@@ -92,3 +92,18 @@ func randomGreeting(s *discordgo.Session, m *discordgo.MessageCreate) (greeting
 
 	return fmt.Sprintf("%s, %s", greetings[0], m.Author.Mention())
 }
+
+// Random coinflip command
+func coinFlip(s *discordgo.Session, m *discordgo.MessageCreate) (coin string) {
```

Comment:
This function isn't called above, on the messageCreate function. The command will not work if the function is not called.